server/s3_option.cc

```diff
@@ -673,6 +673,53 @@ bool S3Option::load_all_sections(bool force_override_from_config = false) {
   return true;
 }
 
+bool S3Option::reload_modifiable_options() {
+  try {
+    YAML::Node root_node = YAML::LoadFile(option_file);
+    if (root_node.IsNull()) {
+      return false;  // File Not Found?
```

Comment:
Do we need to log a message here at error level?
I don't see the caller adding message to log file when YAML::LoadFile() fails here.

Reply:
Added